ethereum/contracts/LightClientBridge.sol

```diff
@@ -83,7 +83,7 @@ contract LightClientBridge {
 
     uint256 public constant THRESHOLD_NUMERATOR = 2;
     uint256 public constant THRESHOLD_DENOMINATOR = 3;
-    uint256 public constant BLOCK_WAIT_PERIOD = 45;
+    uint256 public constant BLOCK_WAIT_PERIOD = 3;
```

Comment:
Shall we create a ticket to reset this to a production valu? Or did we discuss with Web3 Foundation that this should be 3?

Reply:
https://github.com/Snowfork/polkadot-ethereum/issues/426